crates/wasi-http/src/http_impl.rs

```diff
@@ -18,7 +19,15 @@ use tokio::time::timeout;
 #[cfg(not(any(target_arch = "riscv64", target_arch = "s390x")))]
 use tokio_rustls::rustls::{self, OwnedTrustAnchor};
 
-impl crate::default_outgoing_http::Host for WasiHttp {
+fn convert(error: crate::common::Error) -> anyhow::Error {
+    // if let Some(errno) = error.downcast_ref() {
```

Comment:
remove dead code.

Reply:
for this piece, we need to choose which errors we want to throw in place of others